js/src/components/shipping-rate-section/minimum-order-card/changeGroup.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @typedef { import(".~/data/actions").ShippingRate } ShippingRate
+ * @typedef { import("./typedefs").MinimumOrderGroup } MinimumOrderGroup
+ */
```

Comment:
💅  Maybe moving this to` typedefs.js` file?

Reply:
We would still need to import it, like `import("./typedefs").MinimumOrderGroup`